aiotedee/tedee_client.py

```diff
@@ -51,6 +52,7 @@ def __init__(
         timeout: int = TIMEOUT,
         bridge_id: int | None = None,
         session: aiohttp.ClientSession | None = None,
+        api_token_mode_plain: str = API_TOKEN_MODE_PLAIN,
```

Comment:
```suggestion
        api_token_mode_plain: bool = False,
```
Got to be a bool here as well